FsSonarRunner/FsSonarRunner/Program.fs

```diff
@@ -46,8 +46,14 @@ let main argv =
                 let options = XmlHelper.InputXml.Parse(File.ReadAllText(input))
 
                 let metrics = new SQAnalyser()
-                options.Files |> Seq.iter  (fun c -> if File.Exists(c) then metrics.RunAnalyses(c, File.ReadAllText(c), input))
+                let HandleFileToAnalyse(file) = 
+                    if File.Exists(file) then
+                        metrics.RunAnalyses(file, File.ReadAllText(file), input)
+                    else
+                        printf "    [FsSonarRunner] [Error]: %s not found \r\n" file
 
+                options.Files
+                |> Seq.iter (fun file -> HandleFileToAnalyse(file.Replace("file:///", "")))

```

Comment:
I would only call `HandleFileToAnalyse file` here and trim the leading "file:///" with some code like [TrimStart](https://stackoverflow.com/questions/4335878/c-sharp-trimstart-with-string-parameter) inside `HandleFileToAnalyse`.

Shouldn't trimming also done in the case of directory analysis below?

Reply:
nop, that /d is not passed from java side, this is only need because InputFile does not provide filepath only Uri.

Reply:
TrimStart, i will need to copy it a local variable (or doing several times). a bit more code to do the same

Reply:
sure, some more utility code - but what would happen if someone call a file `blahfile:///blah´? The replace would destroy the file name. (Hope noone Comes on such an idea ;) )

Reply:
That one it would work no mather the implementation 😉